exporter/prometheusremotewriteexporter/helper_test.go

```diff
@@ -116,7 +116,7 @@ func TestEnsureTimeseriesPointsAreSortedByTimestamp(t *testing.T) {
 			},
 		},
 	}
-	got := convertTimeseriesToRequest(outOfOrder)
+	got := convertTimeseriesToRequest(outOfOrder, nil)
```

Comment:
There are several combinations of arguments to this function, are they all tested?

Reply:
Replace with a separate method for the metadata requests.